server/root.js

```diff
@@ -100,6 +100,18 @@ let mutations = {
         }
         throw new Error('Invalid username or password');
     },
+    logout: async ({}, { req, res }) => {
+        checkUser(req.user);
+        let token = jwt.sign({
+            id: req.user.id
+        }, 'shouldchangethis', { expiresIn: 0 });
+        res.cookie('jwtAuth', token, { maxAge: 0, httpOnly: true });
```

Comment:
This basically serves to log a user out, as their cookie will be invalid.  Not sure if there's a better way to do it -- seems better than clearing it on the front end somehow

Reply:
I would remove the id field since its not needed either (and when the refactor stuff gets merged add the environment variable for the jwt too)

Reply:
makes sense